library/system/src/lib/yast2/execute.rb

```diff
@@ -33,6 +33,7 @@ class Execute
     Cheetah.default_options = { logger: Y2Logger.instance }
 
     class << self
+      include Yast::I18n
```

Comment:
Does it work? For using `_()` in class methods you have to use `extend` instead of `include`.

This works for me:
```ruby
require "yast"
class Foo
    extend Yast::I18n

    textdomain "foo"

    def self.foo
        puts _("foo")
    end
end

Foo.foo
```